### ISSUE TITLE
How to ignore any exception with Polly?

### ISSUE THREAD

**Issue by moabtools** [App-vNext/Polly#665]
HI!

I have code like this:
```
try
{
    DoSomething();
} catch (Exception) {}
```

How to do it with Polly?
It means I want to try to execute some method and **not** handle any exception when it fails - just keep running.

**Comment by reisenberger** [comment]
Hi @moabtools . If I have understood correctly, you want to swallow exceptions and just keep running, using Polly.

You could create a [FallbackPolicy](https://github.com/App-vNext/Polly/wiki/Fallback) which does nothing. The below example is for a synchronous, void-returning execution (per the example in your question):

    var swallowExceptions = Policy.Handle<Exception>().Fallback(() => { });

    swallowExceptions.Execute(() => DoSomething());

For `TResult`-returning executions, an approach could be similar, but the `FallbackPolicy` would have to specify a substitute `TResult` value.

**Comment by moabtools** [comment]
Hi, @reisenberger. Thanks for reply, but that's not working as expected:

```
static void Main(string[] args)
{
    var swallowExceptions = Policy.Handle<Exception>().Fallback(() => { });
    swallowExceptions.Execute(() => DoSomething());
    Console.ReadLine();
}

static void DoSomething()
{
    throw new Exception(); // VS falls here
}
```
![Image of Yaktocat](http://dl3.joxi.net/drive/2019/07/11/0012/3023/830415/15/458ba958f1.jpg)

But I dont want to handle exceptions at all.

**Comment by reisenberger** [comment]
@moabtools If you run your app without the debugger you will that see it is working perfectly.  

All you are seeing with the debugger, is the debugger breaking when and where the exception is first thrown, to show it to you. When that happens in the debugger, if you press "continue", you will then see the execution proceed, and you will see that the policy does indeed swallow the exception. 

See [this stack overflow question](https://stackoverflow.com/questions/48673173/polly-exception-not-being-caught?noredirect=1&lq=1) and our [detailed wiki article on debugging with Polly](https://github.com/App-vNext/Polly/wiki/Debugging-with-Polly-in-Visual-Studio), for more explanation.

**Comment by moabtools** [comment]
Thats very strange and not transparent. Why not to realize some method like `Policy.Handle<Exception>().NotThrow()` without that debugger dances? This can be very useful in such cases.

**Comment by reisenberger** [comment]
>Why not to realize some method like Policy.Handle<Exception>().NotThrow() without that debugger dances? 

Because it is not possible.  As the article and stackoverflow question I linked explain, this is entirely a debugger artefact. _The way the debugger behaves here is not something that any code within Polly could change._ But we do provide guidance on how to improve your debugging experience. The wiki article I linked explains in detail how to stop these debugger dances, if you don't like them.
